pkgs/development/libraries/librsvg/default.nix

```diff
@@ -31,11 +33,17 @@ stdenv.mkDerivation rec {
     sha256 = "//thsIzVKCqq4UegKzBRZqdCb60iqLlCdwjw8vxCbrw=";
   };
 
+  cargoVendorDir = "vendor";
+
+  strictDeps = true;
+
   nativeBuildInputs = [
+    gdk-pixbuf
     pkg-config
     rustc
     cargo
-    vala
+    rustPlatform.cargoSetupHook
```

Comment:
Hmm, why does it build without this normally?

Reply:
This sets the linker, but Cargo's default linker, `cc`, works fine for non-cross builds.